commands/openers.go

```diff
@@ -55,7 +59,12 @@ var projectLocalOpenCmd = &console.Command{
 		if !pidFile.IsRunning() {
 			return console.Exit("Local web server is down.", 1)
 		}
-		abstractOpenCmd(fmt.Sprintf("%s://127.0.0.1:%d", pidFile.Scheme, pidFile.Port))
+		url := fmt.Sprintf("%s://127.0.0.1:%d", pidFile.Scheme, pidFile.Port)
+		webPath := c.String("webPath")
+		if webPath != "" {
+			url = fmt.Sprintf("%s%s", url, webPath)
```

Comment:
We should probably make sure that a slash is added after the port number, i.e. write something like

```go
fmt.Sprintf("%s/%s", url, strings.TrimLeft(webPath, "/"))
```
(cf. https://pkg.go.dev/strings#TrimLeft)